build-all.sh

```diff
@@ -3,7 +3,7 @@
 set -eu
 
 NODE_STABLE="18"
-NODE_VERSIONS=${NODE_VERSIONS:="20.6.0 18.18.0 16.20.2"}
+NODE_VERSIONS=${NODE_VERSIONS:="20.8.1 18.18.2 16.20.2"}
```

Comment:
We can stop building 16 here as it's end of life so no new versions should get released.